templates/makefile-c++/include/README

```diff
@@ -0,0 +1,39 @@
+
```

Comment:
Consider adding a `.md` extension. Currently this renders on GitHub as plain text but it looks like you are using MD syntax.